functions/graphql/schema/user.js

```diff
@@ -39,7 +40,7 @@ module.exports.resolver = {
     me: (_, __, context) =>
       console.log("CONTEXT", context.user) || context.user,
     user: (_, { id }, context) => {
-      return {};
+      return User.getUserById(id);
```

Comment:
We need to remember to add proper authentication to this query. Not everyone should be able to randomly query users. That can wait for another pull request, though.